bind-service.html.md.erb

```diff
@@ -185,37 +185,42 @@ We need to save the task we get from the text input to the MongoDB. We can do th
             [com.mongodb DB WriteConcern]))
 ```
 
-This snippet of code shows how to import the library `monger` that we will use to access the MongoDB. Now we can implement the function `saveToMongo` that accepts the task as argument. Add it to the bottom of `mongodb.clj`:
+This snippet of code shows how to import the library `monger` that we will use to access the MongoDB. Now we împlement the private function `connectToMongo` to connect to the MongoDB:
```

Comment:
Can you tell the user where to insert the function (in which file and where to put it) it as we did before?